auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Routing.kt

```diff
@@ -252,11 +252,13 @@ fun Route.deltaApiRoutes(
     internalDeltaApiTokenController: InternalDeltaApiTokenController,
 ) {
     route("/delta-api/oauth/token") {
+        install(CORS) {
+            // this is to allow access from Swagger
+            allowHost(Env.getRequiredOrDevFallback("API_ORIGIN", "localhost:8080"))
+            allowHeader("X-Requested-With")
```

Comment:
Did this work? You don't need `allowMethod(HttpMethod.Options`)?

Reply:
Running locally both the proper POST request and the OPTIONS request work.